database/sql/V019__fee_parameters.sql

```diff
@@ -0,0 +1,8 @@
+-- Associate orders with the fee parameters they were evaluated against when they were created for
+-- future debugging.
+CREATE TABLE order_fee_parameters (
```

Comment:
Not related to this PR but in the future it may also be good to store the expected buy amount at order creation time. This will allow to compute how much slippage tolerance the user set and with that allow us to track us exactly how much surplus we generated from batching/CoWs (rather than assuming the price with slippage is the basis for surplus).

I'm not sure if such things would go in a separate table or if this table should become something like "order_metadata"?

Reply:
I don't think we get that as part of the order creation unfortunately :cry:. Also, Its pretty easy to fake when creating an order so may not be super reliable.

Happy to add it though.

Reply:
You mean we don't perform a price estimate when adding the order because most of the time the fee measurement is cached? Yeah maybe it would be nice to add it (could come from some cache and doesn't have to be super up to date)

Reply:
Ah, that is a good point. We can add the estimated amount with the fee measurement we are already storing.